template/src/client/App.js

```diff
@@ -1,6 +1,5 @@
 import React, { Component } from 'react';
 import logo from './logo.svg';
-import './App.css';
```

Comment:
```
Can be solved with new webpack.NormalModuleReplacementPlugin(/\.(css|scss)$/, 'node-noop')
```

That will not work with `css-modules`. You do have alternatives, though:

* https://github.com/css-modules/css-modules-require-hook and 
* https://github.com/janoist1/simple-universal-style-loader